jscomp/napkin/dune

```diff
@@ -0,0 +1,8 @@
+(library
+ (name napkin)
+ (wrapped false)
+ (flags
```

Comment:
since res_syntax is integrated now, we don't need do the copy here, we can just build it as is

Reply:
The issue is that there are currently two different compiler libs in the repo:

1. res_syntax/compiler-libs-406: created by @cristianoc for the syntax repo in https://github.com/rescript-lang/rescript-compiler/pull/5820/files#diff-ae689c9d78337a2d2fdc22773dbd60e8c74cac71ac957d4d6a16a2a9d6fb7c93
2. jscomp/ml: the much more heavily modified one from the compiler repo itself

Currently, the syntax lib, cli and benchmarks (all in res_syntax) are built with 1., but for the compiler we want to build with 2. This is why the files need to be copied.

This behavior is not a change in the current PR, and the compiler libs duplication can be resolved in a future PR.